src/compile/guide.ts

```diff
@@ -2,15 +2,20 @@ import {GuideEncodingEntry} from '../guide';
 import {keys} from '../util';
 import {VgEncodeChannel} from '../vega.schema';
 import {signalOrValueRef} from './common';
-import {wrapCondition} from './mark/encode';
+import {wrapCondition} from './mark/encode/conditional';
 import {UnitModel} from './unit';
 
 export function guideEncodeEntry(encoding: GuideEncodingEntry, model: UnitModel) {
   return keys(encoding).reduce((encode, channel: VgEncodeChannel) => {
-    const valueDef = encoding[channel];
     return {
       ...encode,
-      ...wrapCondition(model, valueDef, channel, def => signalOrValueRef(def.value))
+      ...wrapCondition({
```

Comment:
FYI, no behavior change here. I just switch `wrapCondition` to use parameter argument and make it explicit that `guideEncodeEntry` won't need the new `invalidValueRef` added.